cupy/creation/basic.py

```diff
@@ -134,7 +145,7 @@ def ones(shape, dtype=float):
     return a
 
 
-def ones_like(a, dtype=None, order='K'):
+def ones_like(a, dtype=None, order='K', shape=None):
```

Comment:
To align with the signature `numpy.ones_like(a, dtype=None, order='K', subok=True, shape=None)`, could you insert the `subok` argument?
See e.g. https://github.com/cupy/cupy/blob/fbb745edfad25fa55628ca3571a74781cfb341f2/cupy/core/core.pyx#L351-L352

Reply:
Sure, will do it.

Reply:
Done.